templates/.travis.yml

```diff
@@ -21,6 +21,7 @@ before_install:
 before_script:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
+# if you need tsd  - echo {\"token\":\"$TSDRC_TOKEN\"} > .tsdrc
```

Comment:
btw: probably needs some work since we now should use https://github.com/typings/typings instead of tsd